### ISSUE TITLE
Stack Overflow on windows

### ISSUE THREAD

**Issue by assumptionsoup** [wbond/package_control#1122]
I've tried uninstalling/reinstalling package manager, but this keeps coming back every time I try searching for packages to install.

```
Traceback (most recent call last):
  File "./python3.3/threading.py", line 901, in _bootstrap_inner
  File "package_control.commands.install_package_command in C:\Users\jordan\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 46, in run
  File "package_control.package_installer in C:\Users\jordan\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 59, in make_package_list
  File "package_control.package_manager in C:\Users\jordan\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 453, in list_available_packages
  File "package_control.package_manager in C:\Users\jordan\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 371, in list_repositories
  File "package_control.providers.channel_provider in C:\Users\jordan\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 196, in get_repositories
  File "package_control.providers.channel_provider in C:\Users\jordan\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 87, in fetch
  File "package_control.download_manager in C:\Users\jordan\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 340, in fetch
  File "package_control.downloaders.wininet_downloader in C:\Users\jordan\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 412, in download
  File "package_control.downloaders.wininet_downloader in C:\Users\jordan\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 682, in cache_proxy_info
  File "package_control.downloaders.wininet_downloader in C:\Users\jordan\AppData\Roaming\Sublime Text 3\Installed Packages\Package Control.sublime-package", line 715, in read_option
OSError: exception: stack overflow
```

**Comment by vdovleka** [comment]
I've also seen this exception in read_option method.  

It seems to be related to the fact that "C:\Users\<uname>\AppData\Roaming\Sublime Text 3\Packages\Package Control" directory was missing in my case. After creating it and manually saving the "Package Control.sublime-settings" file ( Preferences->Package Settings->Package Control->Settings Default  +  Save ) the problem seems to be gone.

Just search the following line in your log (Ctrl+`): 

Unable to open /C/Users/<uname>/AppData/Roaming/Sublime Text 3/Packages/Package Control/Package Control.sublime-settings

to see if this is your case as well.

**Comment by wbond** [comment]
Hopefully this should be resolved by 2a041a824d57b1d3f1c3b9b78eea8e02d326c620
